aiida_project/project/venv.py

```diff
@@ -67,3 +71,30 @@ def install_local(self, path):
         install_command.append(Path(self.venv_path, "bin", "pip")).as_posix()
         install_command.extend(["install", "-e", path.as_posix()])
         subprocess.run(install_command, cwd=self.project_path)
+
+
+class UvVenvProject(VenvProject):
+    """An AiiDA environment based on `venv` and `uv`."""
+
+    _engine = "uv"
+    uv_path = UV.as_posix()
+
+    def create(self, python_path: Path) -> None:
+        super().create(python_path)
+        self.venv_path.mkdir(
+            exist_ok=True,
+            parents=True,
+        )
+        venv_command = [self.uv_path, "venv", "-p", f"{python_path.resolve()}", str(self.venv_path)]
```

Comment:
Maybe we can just create the environment with the regular `venv` module? This doesn't take a lot of time, right?